Lachain.CommunicationHub.Net/Hub.cs

```diff
@@ -45,15 +45,20 @@ Lazy<TDelegate> LazyDelegate<TDelegate>()
             );
         }
 
-        public static void Start(string bootstrapAddress)
+        public static void Start(string bootstrapAddress, string peerType)
```

Comment:
the node can change its status,  it can become validator and stop to be validator in different moments,  we should have ability to change its status,  to connect and disconnect to the validator channel, So we need separate method for this.